Vagrantfile

```diff
@@ -26,6 +26,10 @@ CONF = Chassis.config
 
 # Install extensions defined in config file
 Chassis.install_extensions(CONF)
+Chassis.update_core(CONF)
```

Comment:
Might be better to have the check here I was thinking? `Chassis.update_core if CONF['auto_update']['core']`